codec-haproxy/src/test/java/io/netty/handler/codec/haproxy/HAProxyIntegrationTest.java

```diff
@@ -62,13 +62,13 @@ protected void channelRead0(ChannelHandlerContext ctx, HAProxyMessage msg) throw
                   });
               }
           });
-        Channel serverChannel = sb.bind(localAddress).sync().channel();
+        Channel serverChannel = sb.bind(localAddress).sync().getNow();
 
         Bootstrap b = new Bootstrap();
         Channel clientChannel = b.channel(LocalChannel.class)
                                  .handler(HAProxyMessageEncoder.INSTANCE)
                                  .group(group)
-                                 .connect(localAddress).sync().channel();
+                                 .connect(localAddress).sync().getNow();
```

Comment:
I think we could replace a lot of these sync+getNow pattern instances with just `get()`